packages/jmix-front-generator/src/generators/react-typescript/entity-cards-grid/answers.ts

```diff
@@ -41,13 +41,6 @@ const entityCardsGridQuestions: StudioTemplateProperty[] = [
     propertyType: StudioTemplatePropertyType.OPTION,
     required: true,
     options: cardsInRowOptions
-  },
-  {
-    caption: "Entity view",
-    code: "entityView",
-    propertyType: StudioTemplatePropertyType.VIEW,
-    relatedProperty: "entity",
-    required: true
   }
```

Comment:
We need to add a question corresponding to `query` answer. See `displayAttributesQuestions` in `entity-management`.